libnetdata/os.c

```diff
@@ -214,5 +247,4 @@ int getsysctl_by_name(const char *name, void *ptr, size_t len) {
     return 0;
 }
 
-#endif // (TARGET_OS == OS_MACOS)
-
+#endif // (TARGET_OS == OS_MACOS)
```

Comment:
Github is asking for the last line that you removed here, please, bring it back.

Reply:
perfect, I added it on commit v3